### PR TITLE
Added BACPHLIP, inStrain, and geNomad taxonomy to pipeline

### LINKED ISSUE #12: Add bacphlip module to workflow
### Description of feature

Add bacphlip module to workflow

### LINKED ISSUE #13: Add genomad/taxonomy module to workflow
### Description of feature

Add genomad/taxonomy module to workflow

### LINKED ISSUE #15: Add instrain modules to workflow
### Description of feature

Add instrain modules to workflow

### LINKED ISSUE #40: Add prodigal-gv to workflow

### PR DESCRIPTION
**Pull request by CarsonJM**
## PR checklist

- [X] This comment contains a description of changes (with reason).
- [X] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/phageannotator/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/phageannotator _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: modules/local/instrain/stb/main.nf [hunk 1]
```diff
@@ -0,0 +1,49 @@
+process INSTRAIN_STB {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "conda-forge::biopython=1.78 conda-forge::pandas=1.2.4"
```
Comment (sateeshperi) [inline modules/local/instrain/stb/main.nf]: would be consistent if the environment.yml is also used for the local modules as in nf-core modules...agree ?

File: modules/local/instrain/stb/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    tag "modules_local"
+    tag "instrain/stb"
+
+
+    test("['homo_sapiens']['genome']['genome_fasta']") {
+
+        when {
+            params {
+                outdir   = "$outputDir"
+            }
```
Comment (sateeshperi) [inline modules/local/instrain/stb/tests/main.nf.test]: not a big deal...just wanted to point out that you don't need to set this param if not checking published files

File: subworkflows/local/fasta_microdiversity_instrain/tests/main.nf.test [hunk 3]
```diff
@@ -0,0 +1,37 @@
+nextflow_workflow {
+
+    name "Test Subworkflow: FASTA_MICRODIVERSITY_INSTRAIN"
+    script "../main.nf"
+    workflow "FASTA_MICRODIVERSITY_INSTRAIN"
+    tag "subworkflows"
+    tag "subworkflows_local"
+    tag "fasta_microdiversity_instrain"
+    tag "fasta_microdiversity_instrain_default"
```
Comment (sateeshperi) [inline subworkflows/local/fasta_microdiversity_instrain/tests/main.nf.test:9]: need tags for all the modules involved in this sub-wf

**Review by sateeshperi — APPROVED** [review]
minor comments....but rest gtg

File: modules/local/instrain/stb/main.nf (hunk 1 above)
Comment (CarsonJM) [reply modules/local/instrain/stb/main.nf]: Agreed!

**Pull request merged**